### ISSUE TITLE
Emacs org-caldav syncing broken in 18.04

### ISSUE THREAD

**Issue by ddoherty03** [nextcloud/nextcloud-snap#1333]
#### Describe the bug
Since update to 18.04, am getting a 404 error when trying to sync calendar from Emacs using the org-caldav package.

This is a known problem with 18.04 and has been fixed upstream with this pull request to the nextcloud server:

[Only catch anonymous OPTIONS for Office #20632](https://github.com/nextcloud/server/pull/20632/commits/d7161b4eee996554607d0d06da399cd299d8df52)

As discussed [here](https://github.com/dengste/org-caldav/issues/209)

My current versions are 

```
# snap list nextcloud
Name       Version      Rev    Tracking       Publisher   Notes
nextcloud  18.0.4snap1  20498  latest/stable  nextcloud✓  -
```
and:

```
# snap version
snap    2.44.3
snapd   2.44.3
series  16
ubuntu  18.04
kernel  4.15.0-33-generic
```
If I'm understanding this correctly, the fix is due to be in nextcloud-server 18.05.

Any ETA on when the snap will fix this?  Is there a way to manually patch the two lines of code at stake, or revert to a version before the breaking change was made?

Thanks for all your work on the snap, it has been hugely convenient for me.  Regards,

**Comment by kyrofa** [comment]
Indeed, it looks like [the backported version of that fix](https://github.com/nextcloud/server/pull/20680) landed in 18.0.5. Sadly we'll be [skipping 18.0.5 in the snap](https://github.com/nextcloud/nextcloud-snap/issues/1356) due to other issues. You're always welcome to follow the `stable18` daily builds though if you really need this fix and want to live on the edge:

    $ sudo snap refresh nextcloud --channel=18/edge

**Comment by ddoherty03** [comment]
Thanks, @kyrofa.
